docker-compose.yaml

```diff
@@ -1,8 +1,18 @@
 version: "3"
 services:
   aspnet:
-    build:
-      context: .
+    image: wayhowen/devoops:latest
     ports:
-      - "8000:80"
-    command: ls
\ No newline at end of file
+      - "80:80"
+    network:
+      - minitwit
+    restart: always
+    deploy: 
+      replicas: 3
+      update_config:
+        parallelism: 1
+        delay: 10s
```

Comment:
do we actually want the delay

Reply:
a small one at the very least such that if an update should fail we won't have to rollback everything.